File: assets/js/klinker.js
var nextArrow = '<div class="control-next-arrow">▶▶</div>';
var prevArrow = '<div class="control-prev-arrow">◀◀</div>';
var maxSlidePosition = 10;

var control = {
    0: {
        id: "first-promo",
        title: "Клинкерная плитка: предложение",
        prev: "",
        next: "Виды покрытий" + nextArrow,
    },
    1: {
        id: "summary",
        title: "Клинкерная плитка: обзор",
        prev: prevArrow + "Предложение",
        next: "Палитра цветов" + nextArrow,
    },
    2: {
        id: "palitra",
        title: "Клинкерная плитка: палитра цветов",
        prev: prevArrow + "Обзор",
        next: "Качества" + nextArrow,
    },
    3: {
        id: "kachestva",
        title: "Клинкерная плитка: качества",
        prev: prevArrow + "Палитра цветов",
        next: "Технология монтажа" + nextArrow,
    },
    4: {
        id: "schema",
        title: "Клинкерная плитка: монтаж",
        prev: prevArrow + "Качества",
        next: "Устройство откосов" + nextArrow,
    },
    5: {
        id: "otkos",
        title: "Клинкерная плитка: откосы",
        prev: prevArrow + "Технология монтажа",
        next: "Портфолио" + nextArrow,
    },
    6: {
        id: "how-we-work",
        title: "Клинкерная плитка: как мы работаем",
        prev: prevArrow + "Откосы",
        next: "Портфолио" + nextArrow,
    },
    7: {
        id: "portfolio",
        title: "Клинкерная плитка: портфолио",
        prev: prevArrow + "Устройство откосов",
        next: "Видео обзор" + nextArrow,
    },
    8: {
        id: "video",
        title: "Клинкерная плитка: видео обзор",
        prev: prevArrow + "Портфолио",
        next: "Предложение" + nextArrow,
    },
    9: {
        id: "credit-promo",
        title: "Клинкерная плитка: рассрочка",
        prev: prevArrow + "Видео",
        next: "Предложение" + nextArrow,
    },
    10: {
        id: "last-promo",
        title: "Клинкерная плитка: звоните!",
        prev: prevArrow + "Рассрочка",
        next: "",
    },
}

var maxPortfolioPosition = 7;

var portfolio = {
    0: 'url(/assets/img/klinker/portfolio/0.jpg)',
    1: 'url(/assets/img/klinker/portfolio/1.jpg)',
    2: 'url(/assets/img/klinker/portfolio/2.jpg)',
    3: 'url(/assets/img/klinker/portfolio/3.jpg)',
    4: 'url(/assets/img/klinker/portfolio/4.jpg)',
    5: 'url(/assets/img/klinker/portfolio/5.jpg)',
    6: 'url(/assets/img/klinker/portfolio/6.jpg)',
    7: 'url(/assets/img/klinker/portfolio/7.jpg)',
}

File: views/klinker.go
package views

import (
	"html/template"
	"net/http"

	mobiledetect "github.com/Shaked/gomobiledetect"
	"github.com/meehalkoff/loggi"
	"gitlab.com/mvalkov/katzman/lib"
)

func KlinkerHandler(w http.ResponseWriter, r *http.Request) {

	tmpl := template.New("klinker")

	detect := mobiledetect.NewMobileDetect(r, nil)
	if detect.IsMobile() {
		tmpl.ParseFiles(
			"mobile/templates/klinker.html",
			"mobile/templates/header.html",
			"mobile/templates/footer.html",
			"mobile/templates/top-panel.html",
			"mobile/templates/calc.html",
			"mobile/templates/call.html",
			"mobile/templates/about.html",
		)
	} else {
		tmpl.ParseFiles(
			"templates/header.html",
			"templates/footer.html",
			"templates/top-panel/top-panel.html",
			"templates/top-panel/about.html",
			"templates/top-panel/calc.html",
			"templates/top-panel/call.html",
			"templates/top-panel/main-menu.html",
			"templates/control-panel.html",
			"templates/klinker/index.html",
			"templates/firstPromoBlock.html",
			"templates/klinker/summary.html",
			"templates/klinker/palitra.html",
			"templates/klinker/kachestva.html",
			"templates/klinker/schema.html",
			"templates/klinker/otkos.html",
			"templates/klinker/portfolio.html",
			"templates/klinker/video.html",
			"templates/klinker/promo.html",
			"templates/lastPromoBlock.html",
		)
	}
	data := lib.FirstPromoBlock{
		MaterialName: `Клинкерная плитка`,
		ID:           "klinkerPrice",
		PictureURL:   "/assets/img/promo/wide_promo.jpg",
	}
	if err := tmpl.Execute(w, data); err != nil {
		loggi.Fatal(err)
	}
}
